### ISSUE TITLE
Error: Cannot find module "fs"

### ISSUE THREAD

**Issue by jfrux** [bgentry/ember-apollo-client#163]
Something seems up!
Could this be related to me using fastboot?

```
Error: Cannot find module "fs"
    at webpackMissingModule (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:43227:1)
    at Object.<anonymous> (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:43227:1)
    at Object.<anonymous> (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:43304:1)
    at __webpack_require__ (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:20:1)
    at Object.defineProperty.value (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:44563:1)
    at __webpack_require__ (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:20:1)
    at Object.__webpack_exports__.b (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:39714:1)
    at __webpack_require__ (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:20:1)
    at Object.<anonymous> (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:51683:1)
    at __webpack_require__ (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:20:1)
    at /Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:66:1
    at Module.callback (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/-apollo-client-bundle.js:69:1)
    at Module.exports (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:106:1)
    at Module._reify (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:104:1)
    at Module._reify (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:104:1)
    at Module._reify (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:104:1)
    at requireModule (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/vendor/loader/loader.js:27:1)
    at Class._extractDefaultExport (/Users/joshua/Projects/openpilot-configurator-app/tmp/broccoli_merge_trees-output_path-22hU8teG.tmp/assets/addon-tree-output/ember-resolver/resolvers/classic/index.js:408:1)
```

**Comment by bgentry** [comment]
yeah, I just hit this earlier. It's because of the new release of graphql-tools, which is broken in browsers: https://github.com/apollographql/graphql-tools/issues/915

I was hoping they'd fix it fast enough that I wouldn't have to worry about it, but 😞

I think fix would be to edit this project's package.json to also specify that `graphql-tools` has to be `<3.1.0`.

**Comment by bgentry** [comment]
released v1.0.5 that should fix this.
